tests/resources/VCH-Util.robot

```diff
@@ -543,6 +544,27 @@ Cleanup Dangling VMs On Test Server
     \   ${rc}  ${output}=  Run Secret VIC Machine Delete Command  ${vm}
     \   Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  6x  5s  Check Delete Success  ${vm}
 
+Cleanup Dangling Folders On Test Server
+    ${out}=  Run  govc ls vm
```

Comment:
Is `govc ls vm` the best way to do this? We could try `govc find $PATH -type f`

Reply:
This just mirrors the way the other dangling functions work. Find would be better. But these functions are bad and @mhagen-vmware will be removing them in the near future anyways.